CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Fixed
```

Comment:
I'd probably call this an enhancement, not a bug since we just started supporting scoped packages

Reply:
I might have been gaming the system here, angling for this to get done by someone who wasn't me with a sense of urgency. But I decided to do it anyway.